service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -27,23 +27,23 @@
 @Controller
 public class OidcApiController implements OidcApi {
 
-  private final ProviderService providerService;
-  private final LinkedAccountService linkedAccountService;
-  private final SamService samService;
   private final HttpServletRequest request;
+  private final LinkedAccountService linkedAccountService;
   private final ObjectMapper mapper;
+  private final ProviderService providerService;
+  private final SamService samService;
 
   public OidcApiController(
-      ProviderService providerService,
-      LinkedAccountService linkedAccountService,
-      SamService samService,
       HttpServletRequest request,
-      ObjectMapper mapper) {
-    this.providerService = providerService;
-    this.linkedAccountService = linkedAccountService;
-    this.samService = samService;
+      LinkedAccountService linkedAccountService,
+      ObjectMapper mapper,
+      ProviderService providerService,
+      SamService samService) {
     this.request = request;
+    this.linkedAccountService = linkedAccountService;
     this.mapper = mapper;
+    this.providerService = providerService;
+    this.samService = samService;
```

Comment:
why this change?

Reply:
Honestly, I think it may have been accidental. We had been walking through the PR as a group and @zarsky-broad  suggested making these alphabetical, but then we decided it wasn't worth it and went to change it back (but must not have undone all of our changes, hence the new order).

I'd be happy to change it back or make it alphabetical, but I'm pretty sure that's where this weird change came from.

Reply:
no need to revert, I just want to make sure there is no expectation that our parameter lists are alphabetical